modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaScreenContributorClassCheck.testjava

```diff
@@ -0,0 +1,52 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+@Component(service = PortalSettingsConfigurationScreenContributor.class)
+public class UpgradeJavaScreenContributorClassCheck
+	implements PortalSettingsConfigurationScreenContributor {
+
+	@Override
+	public String getCategoryKey() {
+		return "test-category-key";
+	}
+
+	@Override
+	public String getJspPath() {
+		return "/portal_settings/test.jsp";
+	}
+
+	@Override
+	public String getKey() {
+		return "test-key";
+	}
+
+	@Override
+	public String getName(Locale locale) {
+		return _language.get(locale, "test-configuration-name");
+	}
+
+	@Reference
+	private ConfigurationAdmin _configurationAdmin;
+
+	@Reference
+	private ConfigurationProvider _configurationProvider;
+
+	@Reference
+	private Language _language;
+
+	@Reference
+	private OAuth2ApplicationLocalService _oAuth2ApplicationLocalService;
+
+	@Reference
+	private OAuth2AuthorizationLocalService _oAuth2AuthorizationLocalService;
+
+	@Reference(
+		target = "(osgi.web.symbolicname=com.test.web)"
+	)
+	private ServletContext _servletContext;
```

Comment:
@nicolas-sss, I just realized that only the `_language` is being used from the injected dependencies (`@Reference`), therefore there is no need to keep the remaining ones. Let's keep it simple, e.g. remove all attributes using `@Reference` expect the `_language`.